polyglot-kotlin/src/test/resources/unit-tests/artifactId/variation-4/pom.kts

```diff
@@ -0,0 +1,3 @@
+project {
+    id(artifactId = "sample-lib")
```

Comment:
This looks miss-leading, like id of artifact id. WDYT?

Reply:
With just artifactId, it does look a little weird, but it's a unit test for a convenience operation that accepts groupId, artifactId, version, and packaging as separate parameters. The artifactId is the only one that should never be null in this context, so I can set that to required. As it currently stands, the method signature is:
```kotlin
fun id(groupId: String? = null, artifactId: String? = null, version: String? = null, packaging: String? = null) {...}
```

Reply:
Yes, I do understand the reasons, I'm asking about DSL design. Not sure if is such thing as id of the project that includes packaging at least. 

Would be this a better design if it will exists in the scope of `project` just like in java DSL?

Reply:
Didn't know that other DSLs supports the same approach, ok. Make sense to keep it consistent